staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -569,6 +608,40 @@ func (s *sharedIndexInformer) HandleDeltas(obj interface{}) error {
 	return nil
 }
 
+// IsStopped reports whether the informer has already been stopped
+func (s *sharedIndexInformer) IsStopped() bool {
+	s.startedLock.Lock()
+	defer s.startedLock.Unlock()
+	return s.stopped
+}
+
+// RemoveEventHandlerByRegistration tries to remove a formerly added event handler by its
+// registration handle returned for its registration.
+// If a handler has been added multiple times, only the actual registration for the
+// handler will be removed. Other registrations of the same handler will still be
+// active until they are explicitly removes, also.
+func (s *sharedIndexInformer) RemoveEventHandlerByRegistration(handle ResourceEventHandlerRegistration) error {
+	s.startedLock.Lock()
+	defer s.startedLock.Unlock()
+
+	if handle.listener == nil {
```

Comment:
Remove this, we don't set these to nil anymore so this won't optimize anything, right?